build.gradle

```diff
@@ -13,14 +13,18 @@ buildscript {
 
 //add dependency repositories
 repositories {
-	maven {
-		name = "CB Repo"
-		url = "http://chickenbones.net/maven"
-	}
-	maven {
-		name = "JEI Maven"
-		url = "http://dvs1.progwml6.com/files/maven"
-	}
+    maven {
+        name = "CB Repo"
+        url = "http://chickenbones.net/maven"
+    }
+    maven {
+        name = "JEI Maven"
+        url = "http://dvs1.progwml6.com/files/maven"
+    }
+    maven {
+        name = "CurseForge"
+        url = "https://minecraft.curseforge.com/api/maven/"
+    }
```

Comment:
I'm really curious how your build environment is set up without using the build script. Do you have every mod as a module in a monstrous Intellij project? As I was researching various ways to set up projects, I came across something that seems like it would be very useful to someone developing many connected projects simultaneously: [Gradle Composite Builds](https://blog.jetbrains.com/idea/2017/03/webinar-recording-composite-builds-with-gradle/). The key part, I think, is here: [timestamp](https://youtu.be/klk2W02wJ0o?t=1m32s) (up to around 8:15 for the TL;DR, but the whole thing is interesting). The presenter talks about the arduous process of having to upload your changed version to a local maven and then reference the new artifact, which looks like what is going on at the bottom of the build script. Am I completely wrong on this idea, or is this composite build useful? It kinda seems like it would allow a developer to both have easy-to-use dependency management with Gradle as well as a monolithic project structure for rapid development to "edit basically everything on the fly".

Having tons of IDEA modules sounds like the far-monolithic side to get stuff done quick without organization; having a local maven sounds like the far-modular side that's slow but strictly organized; having dependencies replaced in a composite build sounds like a way to get stuff done quick while remaining strictly organized.